src/app/pages/InputPage/InputPage.module.css

```diff
@@ -0,0 +1,12 @@
+.body_div {
```

Comment:
why did you move the overall body settings to only one page and call it body again? 
Maybe just call it ìnputPageContainer`or something not body-isch :)